### DISCUSSION TITLE
Managing the Re-Run in results/reports in allure reports

### DISCUSSION THREAD

**Discussion by akshay-sanganwar** [allure-framework/allure2#1948] [TestOps]
We can trigger re-runs by simple placing the flags like, 

   specFileRetries: 1, 

in browser stack configuration files and trigger the spec file re-runs in case of failures. The resultant report that will be generated should be integrated into the original run  OR a single report in Allure (TestOps). Before few days, this feature was not available but it seems that its been implemented, but there is no link on how to impelkment it.

**Comment by cheshi-mantu** [comment]
@akshay-sanganwar Hello Akshay,
if the rerun on CI side is triggered in the scope of the same pipeline execution, then expected behaviour is that you see the results arriving into the same Launch in Allure TestOps' project and previous results are marked as retried. This feature works like that since the very beginning.

<img width="1610" alt="image" src="https://user-images.githubusercontent.com/8607348/227576969-eb4bf948-6c9d-4dbc-94d6-1e7cbc3afb15.png">

in a test result you'll see the previous test result in the **retries** tab of this test result. 

<img width="1389" alt="image" src="https://user-images.githubusercontent.com/8607348/227577210-0b46ce2c-c0f1-493f-91ac-281a951b5f09.png">
